src/program_generator/constants.py

```diff
@@ -89,3 +89,5 @@
     drop
 """
 EWASM_SOMETHING_LENGTH = 2
+
+MEMORY_OPCODES = ['CALLDATALOAD', 'CALLDATASIZE', 'CALLDATACOPY']
```

Comment:
These are rather `CALLDATA_OPCODES`

Reply:
in this context, their specificity is about being related to memory. Their arguments are treated differently (instead of caring about the bytesize, we care about them themselves). We'll be adding more to this list of similar OPCODEs. I don't want to fragment this unnecessarily.